package.json

```diff
@@ -52,8 +54,10 @@
     "package": "npm run build && mv dist/pulse.xpi addon.xpi",
     "deploy": "deploy-txp",
     "watch": "NODE_ENV=dev webpack --watch",
-    "prettier": "prettier --single-quote --write src/**/*.{js,jsx}",
-    "lint": "eslint . --ext .js --ext .jsx",
+    "prettier": "prettier --single-quote --write 'src/**/*.{js,jsx}'",
+    "lint": "npm-run-all lint:*",
+    "lint:js": "eslint . --ext .js --ext .jsx",
+    "lint:sass": "sass-lint -q -v",
```

Comment:
Can you please add `'src/**/*.scss'` to this to ensure that we're not linting module files?